components/layout_2020/display_list/mod.rs

```diff
@@ -1118,3 +1257,72 @@ fn create_clip_chain(
             .define_clip_chain(parent_clip_chain_id, [new_clip_id]),
     )
 }
+
+/// Compute the additional border-image area.
+pub fn resolve_border_image_outset(
```

Comment:
Do these new `resolve_*` functions need to be `pub`?

Reply:
Nope! I've made them private and have also added more documentation here.